### DISCUSSION TITLE
Published GitHub pages stopped opening since April

### DISCUSSION THREAD

**Discussion by TopmanTech** [community/community#21580] [Other Feature Feedback, Questions, & Ideas]
<p>Hello Please who else is facing this issue and already found solution . For almost 2 years , the websites I published on GitHub pages was working fine until recently … It stopped and I’ve tried looking for the cause and how to get it working again BUT I COULDN’T. is it like github stoped the service permanently or what  ?</p>
<p>I also tried to follow the “what action should I take” in this article below and still I couldn’t get it to work. Pls help.</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.blog/wp-content/uploads/2019/01/cropped-github-favicon-512.png?fit=32%2C32" class="site-icon" width="32" height="32">

      <a href="https://github.blog/changelog/2021-01-29-github-pages-will-stop-redirecting-pages-sites-from-github-com-after-april-15-2021/#:~:text=github.com%20,%20after%20April%2015,github.com%20" target="_blank" rel="noopener nofollow ugc">The GitHub Blog</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://github.blog/wp-content/uploads/2019/03/blog-card.png?fit=5001%2C2626" class="thumbnail" width="690" height="362"></div>

<h3><a href="https://github.blog/changelog/2021-01-29-github-pages-will-stop-redirecting-pages-sites-from-github-com-after-april-15-2021/#:~:text=github.com%20,%20after%20April%2015,github.com%20" target="_blank" rel="noopener nofollow ugc">GitHub Pages will stop redirecting Pages sites from *.github.com after April...</a></h3>

  <p>GitHub Pages will stop redirecting Pages sites from *.github.com after April 15, 2021</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by tcbyrd** [comment, marked as answer]
<p>Welcome <a class="mention" href="/topmantech">@TopmanTech</a>! Let me see if I can clarify for you.</p>
<blockquote>
<p>is it like github stopped the service permanently…</p>
</blockquote>
<p>The change in the referenced blog post is that we’ve stopped <strong>redirecting</strong> GitHub Pages sites from <code>*.github.com</code>. Any sites you had on Pages still work on <code>*.github.io</code>.</p>
<p>As an example, before that change if you typed in <code>twitter.github.com</code>, it would automatically redirect to <code>twitter.github.io</code>. Now you have to go to <code>twitter.github.io</code> directly, as <code>twitter.github.com</code> no longer redirects you automatically.</p>
<p>Nothing was deleted and the service is still running. In most cases all you have to do is update your URLs from <code>&lt;user&gt;.github.com</code> to <code>&lt;user&gt;.github.io</code>.</p>
<p>Hope that makes sense!</p>

**Comment by TopmanTech** [comment]
<p>Yes it does. Changed some links and it now works. Thanks :grin:</p>

**Comment by shameer1919** [comment]
My account hacked please help
